axe.d.ts

```diff
@@ -1,6 +1,5 @@
 // Type definitions for axe-core
 // Project: https://github.com/dequelabs/axe-core
-// Definitions by: Marcy Sutton <https://github.com/marcysutton>
```

Comment:
For the record.  Removing this because Marcy should not be the person someone reaches out to for issues with this file as this comment might suggest. No other code file in axe-core includes info on who did the initial commit. It has been updated and expanded on since then many times. I'm removing this line because its out of place.

Reply:
The other past employees should also be removed from `package.json`.